docs/resources/deployment-examples/example-k8s/anchor-platform-dev/reference-server-config-map.yaml

```diff
@@ -23,8 +23,8 @@ data:
 
     integration-auth:
       authType: none
-      platformToAnchorSecret: myPlatformToAnchorSecret
-      anchorToPlatformSecret: myAnchorToPlatformSecret
+      platformToAnchorSecret: myPlatformToAnchorSecret1234567890
```

Comment:
Trailing `1234567890`?

Reply:
It needs to be 256 bit, all key changes are necessary